js/coinflex.js

```diff
@@ -1460,10 +1460,7 @@ module.exports = class coinflex extends Exchange {
         } else {
             finalLimitPrice = limitPrice;
         }
-        let trades = this.safeValue (order, 'matchIds');
```

Comment:
@ttodua FYI: I've committed a small fix to this, the parsing is done inside safeOrder so we can't do it here

Reply:
ahh, right !